test/ttTransactionTestFiller.json

```diff
@@ -241,21 +241,6 @@
         }
     },
 
-    "WrongAddress" : {
```

Comment:
because when we try to read this wrong address from json filler, an exception is thrown not producing a test rlp and it makes a failure
instead I created wrong transaction rlps manually in ttWrongRLPTransaction.json




Четверг, 19 февраля 2015, 4:27 -08:00 от Christoph Jentzsch <notifications@github.com>:
>In  test/ttTransactionTestFiller.json :
>> @@ -241,21 +241,6 @@
>          }
>      },
>  
> -    "WrongAddress" : {
>Why do you remove this testFiller?
>—
>Reply to this email directly or  view it on GitHub .


--------------
Wins Vega Wins